inputs/costs_PyPSA.csv

```diff
@@ -218,3 +218,6 @@ offwind-ac-connection-underground,2030,investment,1342,EUR/MW/km,DEA https://ens
 offwind-dc-station,2030,investment,400,EUR/kWel,Haertel 2017; assuming one onshore and one offshore node + 13% learning reduction
 offwind-dc-connection-submarine,2030,investment,2000,EUR/MW/km,DTU report based on Fig 34 of https://ec.europa.eu/energy/sites/ener/files/documents/2014_nsog_report.pdf
 offwind-dc-connection-underground,2030,investment,1000,EUR/MW/km,Haertel 2017; average + 13% learning reduction
+offwind-float-station,2030,investment,400,EUR/kWel,Haertel 2017; assuming one onshore and one offshore node + 13% learning reduction
+offwind-float-connection-submarine,2030,investment,2000,EUR/MW/km,DTU report based on Fig 34 of https://ec.europa.eu/energy/sites/ener/files/documents/2014_nsog_report.pdf
+offwind-float-connection-underground,2030,investment,1000,EUR/MW/km,Haertel 2017; average + 13% learning reduction
```

Comment:
add those in `inputs/manual_input.csv`, this file is only legacy and shouldn't be updated.

Reply:
okay, I moved them to manual_inputs.